python_scripts/metrics.py

```diff
@@ -586,10 +597,10 @@ def plot_pr_roc_interactive():
 
 # %% [markdown]
 # While some features are numeric, some have been tagged as `category`. These
-# features need to be encoded in a proper way such that our random forest can
+# features need to be encoded such that our random forest can
```

Comment:
Maybe expand that generally all (all sklearn estimators at least) need numerical data and cannot work with string data.